docs/content/mobile/android/local-deploy.md

```diff
@@ -1,73 +1,69 @@
-
-  1. Play the game on the Emulator.
+1. In the Unity Editor, navigate to **SpatialOS** > **Build for local**. Select your android worker, and wait for the build to complete.
+1. Click **SpatialOS** > **Launch mobile client** > **Android Device**.
+1. Play the game on the Emulator.
 
 ## Connecting your Android device to a local deployment
 
-  1. Enable USB debugging on your mobile device. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
-  1. Make sure your computer and your mobile device are both connected to the same wireless network.
-  1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
-  1. [Build your server-workers.]({{urlRoot}}/content/build)
-  1. You need to know the local IP address of your computer to connect. [This page (on the Lifehacker website)](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address) describes how you can find your local and external IP address.
-  1. In a terminal window from the root folder of your SpatialOS project,  run: `spatial local launch mobile_launch.json --runtime_ip=<your-local-ip>`. (Where `<your-local-ip>` is the IP address you just located.)
+1. Enable USB debugging on your mobile device. See the [Android developer documentation](https://developer.android.com/studio/debug/dev-options#enable) for guidance.
+1. Make sure your computer and your mobile device are both connected to the same wireless network.
+1. Connect the mobile device to your computer using a USB cable. You might get a pop-up window on the device asking you to allow USB debugging. Select **Yes**.
+1. [Setup your local deployment](#setup-local-deployment-for-mobile-phone-connections)
+1. [Build your server-workers.]({{urlRoot}}/content/build)
+1. Start the local deployment from the Unity Editor though **SpatialOS** > **Local launch**.
 
-    > You cannot use **SpatialOS** > **Local launch** in your Unity Editor, because you need to specify the runtime IP and a different launch configuration.
-    >
     > **It’s done when:** You see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`
 
-  1. In the Unity Editor, go to **File** > **Build Settings**. Choose the mobile device that you connected to your computer from the drop-down menu in the **Run Device** field.
-  1. Click **Build and Run**.
-
-    > For subsequent runs you can just select **File** > **Build and Run**.
-
-  1. Once the game is running on your device, you see an empty text field and a **Connect** button: enter the local IP address of your computer in the text field and click **Connect**.
-  1. Play the game on your mobile device.
+1. In the Unity Editor, navigate to **SpatialOS** > **Build for local**. Select your android worker, and wait for the build to complete.
```

Comment:
"through" instead of "though"

"your local deployment" instead of "the local deployment" -> just makes it a bit clearer that this is happening on their side